site/content/docs/main/locations.md

```diff
@@ -166,6 +176,50 @@ During backup creation:
 velero backup create full-cluster-backup
 ```
 
+### Create a storage location that uses unique credentials
+
+It is possible to create additional `BackupStorageLocations` that use their own credentials.
+This enables you to save backups to another storage provider or to another account with the storage provider you are already using.
+
+If you create additional `BackupStorageLocations` without specifying the credentials to use, Velero will use the credentials provided at install time and stored in the `cloud-credentials` secret.
+Please see the [earlier example][9] for details on how to create multiple `BackupStorageLocations` that use the same credentials.
+
+#### Prerequisites
+- This feature requires support from the [object storage provider plugin][5] you wish to use.
+  All plugins provided by the Velero team support this feature.
+  If you are using a plugin from another provider, please check their documentation to determine if this is supported.
+- The [plugin for the object storage provider][5] you wish to use must be [installed][6].
+- You must create a file with the object storage credentials. Follow the instructions provided by your object storage provider plugin to create this file.
+
+Once you have installed the necessary plugin and created the credentials file, create a [Kubernetes Secret][6] in the Velero namespace that contains these credentials:
+
+```shell
+kubectl create secret generic -n velero credentials --from-file=bsl=</path/to/credentialsfile>
+```
+
+This will create a secret named `credentials` with a single key (`bsl`) which contains the contents of your credentials file.
+The name and key of this secret will be given to Velero when creating the `BackupStorageLocation` using the flag `--credential`.
```

Comment:
I think we can be more direct/specific here, so maybe something like, "Next, create a `BackupStorageLocation` that uses this Secret by passing the Secret name in the `--credential` flag. When interacting with a this `BackupStroageLocation` in the future, Velero will fetch the data from the key within the Secret you provide."